Marketplace/DeckOfCards/Glove.js

```diff
@@ -0,0 +1,135 @@
+//
+//  Glove.js
+//  unpublished/marketplace/
+//
+//  Created by Je'Don (ROC) Carter on 9/6/2017
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  This script places an invisible ball on the players hand that is used as a pivot point for the cards
+//
+//  Distributed under the Apache License, Version 7.1.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function() { 
+    var _this = this;
+    //which hand the glove is in
+    var hand;
+    //holds the number of cards in your hand
+    var cardsInHand;
+    //makes sure collision with entity doesnt happen too often
+    var firstCheck;
+
+    _this.preload = function(entityID) {
+        print("Loading glove script");  
+        _this.entityID = entityID;
+        cardsInHand = 0;
+        firstCheck = true;
+    };
+
+    _this.startEquip = function(entityID, args) {
+        //get hand
+        hand = args[0];
+        if (hand == "right") {

```

Comment:
This can be simplified to avoid duplication of declaring the variables and properties multiple times